File: app/src/main/java/com/synapse/social/studioasinc/widget/ZoomImageViewLib/gestures/OnScaleAndMoveGestureListener.java
package com.synapse.social.studioasinc.widget.ZoomImageViewLib.gestures;

import android.view.ScaleGestureDetector;

public interface OnScaleAndMoveGestureListener {
    void onScaleAndMove(ScaleGestureDetector scaleGestureDetector, float f, float f2, float f3);

    void onScaleBegin(ScaleGestureDetector scaleGestureDetector);

    void onScaleEnd(ScaleGestureDetector scaleGestureDetector, float f, float f2);
}

File: app/src/main/java/com/synapse/social/studioasinc/widget/ZoomImageViewLib/gestures/OnScaleAndMoveGestureListener.kt
package com.synapse.social.studioasinc.widget.ZoomImageViewLib.gestures

import android.view.ScaleGestureDetector

interface OnScaleAndMoveGestureListener {
    fun onScaleAndMove(scaleGestureDetector: ScaleGestureDetector, currentScale: Float, moveDistanceFromX: Float, moveDistanceFromY: Float)
    fun onScaleBegin(scaleGestureDetector: ScaleGestureDetector)
    fun onScaleEnd(scaleGestureDetector: ScaleGestureDetector, moveDistanceFromX: Float, moveDistanceFromY: Float)
}
